src/coins.h

```diff
@@ -184,6 +184,7 @@ struct CCoinsCacheEntry
     inline bool IsDirty() const noexcept { return m_flags & DIRTY; }
     inline bool IsFresh() const noexcept { return m_flags & FRESH; }
     inline CoinsCachePair* Next() const noexcept { return m_next; }
+    inline CoinsCachePair* Prev() const noexcept { return m_prev; }
```

Comment:
Now that we have this, could you please cover it with tests as well (where we're testing that the linked list is constructed correctly)?
Or would it make more sense to just call `SanityCheck()`since it's already checking the linked list's integrity?

<img src="https://github.com/user-attachments/assets/782fc1a0-8d78-4ad2-a963-8e77337a8a39">

Reply:
Hmm this was added by the cherry-picked commit from @sipa. Can we leave it for a follow-up?

Reply:
Of course

Reply:
I can add Prev tests to the coinscachepair_tests and SanityCheck to the coins_tests in a follow-up.

Reply:
Added tests.